Makefile

```diff
@@ -0,0 +1,18 @@
+.DEFAULT_GOAL := help
+
+.PHONY: start
+start: ## Start the development environment (use Docker)
+	$(SUDO) docker run \
+		--rm \
+		-v $(shell pwd):/var/www/FreshRSS:z \
```

Comment:
I think you could take advantage of https://github.com/FreshRSS/FreshRSS/pull/2508 and use `-e FRESHRSS_ENV=development`

Reply:
I started to add it and it came to my mind a developer still can change its `data/config.php` file and might want to test FRSS in production mode. I agree it would be a minority of use cases and it's better for developers to start immediately in dev mode, so I'll probably set the environment variable, but I wanted to share my thoughts first ;)